elyra/pipeline/processor_airflow.py

```diff
@@ -39,30 +38,25 @@ def type(self):
         return self._type
 
     def process(self, pipeline):
-        timestamp = datetime.now().strftime("%m%d%H%M%S")
-        pipeline_name = (pipeline.title if pipeline.title else 'pipeline') + '-' + timestamp
+        unique_id = uuid.uuid4().hex[4:15]
+        pipeline.title = pipeline.title + '-' + unique_id
```

Comment:
The previous code only applied the uniqueness if title wasn't set.  This makes it unconditional and, if not set, will result in a title like '-9347a3c323f'.  Does pipeline.title always have a value prior to this code?

I like the idea of applying uniqueness unconditionally, just want to be sure if we still need the check for title not having a value or not.